flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Any reason for only allowing x86_64, there is a i386 binary available: https://teamspeak.com/en/downloads/

Reply:
I need to download it in extra data and didn't find a way how to download only the correct one. It always downloaded both of them.
Even if I used `only-arches` in sources.

Reply:
```
diff --git a/com.teamspeak.TeamSpeak.yaml b/com.teamspeak.TeamSpeak.yaml
index d800ed3..e050ed1 100644
--- a/com.teamspeak.TeamSpeak.yaml
+++ b/com.teamspeak.TeamSpeak.yaml
@@ -30,6 +30,15 @@ modules:
       - type: file
         path: com.teamspeak.TeamSpeak.appdata.xml
       - type: extra-data
+        only-arches:
+          - i386
+        filename: TeamSpeak3-Client.run
+        url: http://files.teamspeak-services.com/releases/client/3.2.3/TeamSpeak3-Client-linux_x86-3.2.3.run
+        sha256: 97735cae8120b8fda49846ffe73010d9da7b82f2a2805387cd203a62f1c8a8c7
+        size: 75997823
+      - type: extra-data
+        only-arches:
+          - x86_64
         filename: TeamSpeak3-Client.run
         url: https://files.teamspeak-services.com/releases/client/3.2.3/TeamSpeak3-Client-linux_amd64-3.2.3.run
         sha256: 3fc60446a19cc0ae4c26506c28f4ab3dc4eb8db75f573496a8f6c71d712df085
```

This seems to work fine.

Reply:
Hm, it works. I tried the same, but maybe I used incorrect syntax.

Reply:
I added a new commit, this should be now solved.